bodhi/server/util.py

```diff
@@ -684,3 +684,35 @@ def sort_severity(value):
     }
 
     return value_map.get(value, 99)
+
+
+def infer_content_class(base, build):
+    """ Given a base class and a build from koji, identify and return the child
+    class associated with the appropriate ContentType.
+
+    For example, given the Package base class and a normal koji build, return
+    the RpmPackage model class. Or, given the Build base class and a container
+    build, return the ContainerBuild model class.
+
+    Arguments:
+    base -- A base model class.
+    build -- A dict of information from the build system (koji).
+    """
+
+    # Import this here to avoid circular import
+    from bodhi.server.models import ContentType
```

Comment:
Circular imports are usually a sign that there is an organization problem. I.e., if this module uses the models class, and the models class uses this module, then these things should probably be the same. In other words, we should move this function to the models module.